doc/specs/#1043 - Set the initial position of the Terminal/spec.md

```diff
@@ -0,0 +1,103 @@
+---

```

Comment:
This file should be removed from this change, since it is part of the separate spec PR, right?

Reply:
Yes, please do not include the spec in both PRs.

Reply:
removed